### DISCUSSION TITLE
Home thermostat BHT - 8000

### DISCUSSION THREAD

**Discussion by GjelbrimH** [make-all/tuya-local#1451] [Q&A]
Hello to everyone. 
I bought the BHT - 8000 WIFI thermostat 2 years ago, so it is wireless and everything works fine except for the temperature when it should be turned on. 
The problem is that if I set the temperature to 23C or whatever temperature, the moment the temperature in the room drops to 22.5C, the boiler turns on, so only for 0.5C. For me, it is too little, the 1C is perfect, so I'd love to work this way like when I set it to 23C (or whatever temperature) and when it drops to 22C, then the heating can turn on (fire). I have also tried the dead zone temp function, but if I do it then it rises to 1.5C and I can't make it 1C anyway.
![photobank-5](https://github.com/make-all/tuya-local/assets/154633644/0116cf23-31fe-4f9c-9524-77d16d12c5f9)

Please, who can help me?

**Comment by make-all** [comment]
You'd have to  download the full dps list from iot.tuya.com (QueryThingsDataModel API call) to see what is available on the device.
